cosmwasm/packages/wasmi-runtime/src/wasm/runtime/contract.rs

```diff
@@ -628,6 +628,12 @@ impl WasmiApi for ContractInstance {
             Ok(x) => x,
         };
 
+        if human_addr_str.is_empty() {
+            return Ok(Some(RuntimeValue::I32(
+                self.write_to_memory(b"input is empty")? as i32,
+            )));
+        }
+
```

Comment:
`addr_validate` is a new API therefore this check is not necessary, please remove this.

Reply:
It is necessary though:
https://github.com/CosmWasm/cosmwasm/blob/main/packages/vm/src/imports.rs#L129

Reply:
We never looked at these, so there's no reason to start now.